publications-2019.bib

```diff
@@ -1,5 +1,222 @@
+  year={2019},
+  publisher={Universit{\"a}t zu K{\"o}ln}
+}
+
+@article{EbnaHai2019,
+  doi = {10.1016/j.apm.2019.07.007},
+  url = {https://doi.org/10.1016/j.apm.2019.07.007},
+  year = {2019},
+  month = nov,
+  publisher = {Elsevier {BV}},
+  volume = {75},
+  pages = {916--939},
+  author = {Bhuiyan Shameem Mahmood Ebna Hai and Markus Bause and Paul Allen Kuberry},
+  title = {Modeling concept and numerical simulation of ultrasonic wave propagation in a moving fluid-structure domain based on a monolithic approach},
+  journal = {Applied Mathematical Modelling}
+}
+
+@article{Hagstrom2019,
+  doi = {10.1007/s00211-018-1012-0},
+  url = {https://doi.org/10.1007/s00211-018-1012-0},
+  year = {2019},
+  month = jan,
+  publisher = {Springer Science and Business Media {LLC}},
+  volume = {141},
+  number = {4},
+  pages = {917--966},
+  author = {Thomas Hagstrom and Seungil Kim},
+  title = {Complete radiation boundary conditions for the Helmholtz equation I: waveguides},
+  journal = {Numerische Mathematik}
+}
+
+@article{Guo2019,
+  doi = {10.2118/194493-pa},
+  url = {https://doi.org/10.2118/194493-pa},
+  year = {2019},
+  month = feb,
+  publisher = {Society of Petroleum Engineers ({SPE})},
+  volume = {22},
+  number = {03},
+  pages = {842--860},
+  author = {X.. Guo and K.. Wu and J.. Killough and J.. Tang},
+  title = {Understanding the Mechanism of Interwell Fracturing Interference With Reservoir/Geomechanics/Fracturing Modeling in Eagle Ford Shale},
+  journal = {{SPE} Reservoir Evaluation {\&} Engineering}
+}
+
+@article{Gulevich2019,
+  doi = {10.1103/physrevb.99.060501},
+  url = {https://doi.org/10.1103/physrevb.99.060501},
+  year = {2019},
+  month = feb,
+  publisher = {American Physical Society ({APS})},
+  volume = {99},
+  number = {6},
+  author = {D. R. Gulevich and V. P. Koshelets and F.~V. Kusmartsev},
+  title = {Bridging the terahertz gap for chaotic sources with superconducting junctions},
+  journal = {Physical Review B}
+}
+
+@article{German2019,
+  doi = {10.1016/j.anucene.2019.05.049},
+  url = {https://doi.org/10.1016/j.anucene.2019.05.049},
+  year = {2019},
+  month = dec,
+  publisher = {Elsevier {BV}},
+  volume = {134},
+  pages = {144--157},
+  author = {P{\'{e}}ter German and Jean C. Ragusa},
+  title = {Reduced-order modeling of parameterized multi-group diffusion k-eigenvalue problems},
+  journal = {Annals of Nuclear Energy}
+}
+
+@phdthesis{gerken2019dynamic,
+  title={Dynamic Inverse Problems for Wave Phenomena},
+  author={Gerken, Thies},
+  year={2019},
+  school={Universit{\"a}t Bremen}
+}
+
+@article{https://doi.org/10.13140/rg.2.2.18015.41123,
+  doi = {10.13140/RG.2.2.18015.41123},
+  url = {http://rgdoi.net/10.13140/RG.2.2.18015.41123},
+  author = {Exner,  Pavel},
+  language = {en},
+  title = {Extended finite element method for approximation of singularities in groundwater flow},
+  publisher = {Unpublished},
+  year = {2019}
+}
```

Comment:
This is a thesis, not an article.

Reply:
Maybe use the university library link for this, instead of the doi which only points to researchgate?
 https://dspace.tul.cz/handle/15240/153160